addons/sale_timesheet/static/tests/tours/sale_timesheet_tour.js

```diff
@@ -41,9 +41,6 @@ registry.category("web_tour.tours").add('sale_timesheet_tour', {
     trigger: '.o_field_widget.o_project_name input',
     content: 'Select your project name (e.g. Project for Freeman)',
     run: "edit Project for Freeman",
-}, {
-    trigger: 'div[name="allow_billable"] input',
-    run: 'click',
```

Comment:
This step set the project as billable. Now it is the default, so I had to remove it. Otherwise, it turns non-billable